### ISSUE TITLE
USWDS - UX: Verbose Time Picker instructions for clearing content

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds#6151]
1 blind user was confused by the 'clear select contents' announcement and said that it would be clearer to him if it announced something more specific like "clear the time."

## Next steps

1. Create a feature branch for use in the UX research prototype
1. Change announcement copy `clear select contents → clear selection`
1. Create a draft PR clearly marked it's **not** meant for merge

**Comment by jaclinec** [comment]
We discussed usability testing planning in yesterday's CS UX sync, and determined that we need to develop the time picker's "clear selection" announcement so that we can test it in the next round of usability testing. Changes will not be merged, they will only be made for the prototype. 

Please move forward with implementing a time picker announcement of "clear selection" (to replace "clear select contents").

**Comment by dct2023** [comment]
@uswds/feds Can you all review this issue and confirm if this should be prioritized for March? If not, we can add it to the backlog and then review against others.

**Comment by jaclinec** [comment]
I just wanted to chime in and mention that so far, the "clear selection" language is testing really well. We've tested it with 3 screen reader users so far, hoping to test with a couple more. Unless we get wildly different results in the next couple of sessions, it's probably safe to say we can move forward with merging these changes.

I will update this issue with final results when testing is complete.

**Comment by annepetersen** [comment]
This doesn't feel like a huge lift since it's just a text change (correct me if I'm wrong, y'all!), but it makes a big difference, so I'd tend toward recommending it be prioritized for March.
